lib/ui/text.dart

```diff
@@ -1445,6 +1490,10 @@ class ParagraphBuilder extends NativeFieldWrapperClass2 {
     String locale
   ) native 'ParagraphBuilder_constructor';
 
+  int _placeholderCount;
+  /// The number of placeholders currently in the paragraph.
+  int get placeholderCount => _placeholderCount;
```

Comment:
trivial nit: we put the getter before the field according to https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#order-other-class-members-in-a-way-that-makes-sense